libbeat/docs/shared-deduplication.asciidoc

```diff
@@ -0,0 +1,109 @@
+[id="{beatname_lc}-deduplication"]
+== Data deduplication
+
+The {beats} framework guarantees at-least-once delivery to ensure that no data
+is lost when events are sent to {es}. This is great if everything goes as
```

Comment:
Besides Elasticsearch there are other outputs that support acknowledgement as well, notably Logstash, Kafka, and Redis. Do we want to mention these here for completeness and then focus on Elasticsearch as the `_id` field has a special meaning to it?